packages/compiler/src/server/language-config.ts

```diff
@@ -99,9 +99,3 @@ export const TypeSpecLanguageConfiguration = {
     },
   ],
 } as const;
-
-/**
- * @deprecated Use TypeSpecLanguageConfiguration
- * @hidden
- */
-export const CadlLanguageConfiguration = TypeSpecLanguageConfiguration;
```

Comment:
I think there is still quite a few other things referencing cadl we should remove